app/views/markets/mkt_home.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1> Welcome, <%= @market.name %> </h1>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Address</th>
+    <th>City</th>
+    <th>County</th>
+    <th>State</th>
+    <th>Zip</th>
+    <th>Vendors</th>
+  </tr>
+
+  <tr>
+    <td><%= @market.name %></td>
+    <td><%= @market.address %></td>
+    <td><%= @market.city %></td>
+    <td><%= @market.country %></td>
+    <td><%= @market.state %></td>
+    <td><%= @market.zip %></td>
+    <td>
+      <ul>
+        <% @market.vendors.each do |vendor| %>
+        <li><%= vendor.name %> <%= link_to "Delete", vendor, method: :delete %></li>
+        <% end %>
+      </ul>
+    </td>
+  </tr>
+</table>
+
+<section class ="footer">
+<%= link_to "Create a New Market", new_market_path %>
+
+<%= link_to "Edit Yourself", edit_market_path(:id => @market.id) %>
+<%= link_to "Add a New Vendor", new_market_vendor_path(:market_id => @market.id) %>
```

Comment:
This and the line above could be simplified a bit by changing, for example, `:id => @market.id` to just `@market`. Rails is smart enough to know that (a) we want to use the ID value from our Market, and (b) that value should be put where `:id` or `:market_id` is in the path.